packages/playground/cli/src/run-cli.ts

```diff
@@ -338,16 +389,86 @@ export async function runCLI(args: RunCLIArgs): Promise<RunCLIServer> {
 				} else if (args.command === 'run-blueprint') {
 					logger.log(`Blueprint executed`);
 					process.exit(0);
-				} else {
-					logger.log(`WordPress is running on ${absoluteUrl}`);
 				}
 
-				return { requestHandler, server };
+				if (
+					args.experimentalMultiWorker &&
+					args.experimentalMultiWorker > 1
+				) {
+					logger.log(`Preparing additional workers...`);
+
+					// Save /internal directory from initial worker so we can replicate it
+					// in each additional worker.
+					const internalZip = await zipDirectory(
+						playground,
+						'/internal'
+					);
+
+					// Boot additional workers
+					const initialWorkerProcessIdSpace = processIdSpaceLength;
+					await Promise.all(
+						additionalWorkers.map(async (worker, index) => {
+							const additionalPlayground =
+								consumeAPI<PlaygroundCliWorker>(worker);
+							await additionalPlayground.isConnected();
+							exposeAPI(fileLockManager, undefined, worker);
+
+							const firstProcessId =
+								initialWorkerProcessIdSpace +
+								index * processIdSpaceLength;
+
+							await additionalPlayground.boot({
+								phpVersion: compiledBlueprint.versions.php,
+								absoluteUrl,
+								mountsBeforeWpInstall,
+								mountsAfterWpInstall,
+								// Skip WordPress zip because we share the /wordpress directory
+								// populated by the initial worker.
+								wordPressZip: undefined,
+								// Skip SQLite integration plugin for now because we
+								// will copy it from primary's `/internal` directory.
+								sqliteIntegrationPluginZip: undefined,
+								dataSqlPath:
+									'/wordpress/wp-content/database/.ht.sqlite',
+								firstProcessId,
+								processIdSpaceLength,
+								followSymlinks,
+								trace,
+							});
+							await additionalPlayground.isReady();
+
+							// Replicate the Blueprint-initialized /internal directory
+							await additionalPlayground.writeFile(
+								'/tmp/internal.zip',
+								internalZip
+							);
+							await unzipFile(
+								additionalPlayground,
+								'/tmp/internal.zip',
+								'/internal'
+							);
+							await additionalPlayground.unlink(
+								'/tmp/internal.zip'
+							);
+
+							loadBalancer.addWorker(additionalPlayground);
+						})
+					);
+
+					logger.log(`Ready!`);
+				}
+
+				logger.log(`WordPress is running on ${absoluteUrl}`);
+
+				return {
+					playground,
+					server,
+				};
 			} catch (error) {
 				if (!args.debug) {
 					throw error;
 				}
-				const phpLogs = php.readFileAsText(errorLogPath);
+				const phpLogs = await playground.readFileAsText(errorLogPath);
```

Comment:
Do we need to get the error log from a specific worker? Or do they all append to the same error log file? Any risk of conflicting writes?